source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/list/my-projects.tsx

```diff
@@ -36,9 +36,10 @@ export default compose(
     };
   }),
   withNetwork(),
+  withCache( () =>  ( { projects :  q => q.findRecords( PROJECT ) }) ),
   withLoader(({ error, projects }) => !error && !projects),
   withProps(({ projects }) => ({
-    projects: projects.filter(resource => resource.type === PROJECT),
+    projects: projects.filter(resource => resource.type === PROJECT && resource.attributes.dateArchived == null),
```

Comment:
ez :)

Reply:
what is ez?

Reply:
this way of filtering out archived things :)